source/MRMesh/MRGcodeProcessor.h

```diff
@@ -160,8 +155,9 @@ class MRMESH_CLASS GcodeProcessor
 
     // internal / machine settings
     float accuracy_ = 1.e-3f;
-    std::array<Vector3f, 3> rotationAxes_ = { Vector3f::minusX(), Vector3f::minusY(), Vector3f::plusZ() };
-    RotationAxisOrder rotationAxesOrder_ = { RotationAxisName::A, RotationAxisName::B, RotationAxisName::C };
+//     std::array<Vector3f, 3> rotationAxes_ = { Vector3f::minusX(), Vector3f::minusY(), Vector3f::plusZ() };
+//     CNCMachineSettings::RotationAxisOrder rotationAxesOrder_ = { CNCMachineSettings::RotationAxisName::A, CNCMachineSettings::RotationAxisName::B, CNCMachineSettings::RotationAxisName::C };
+    CNCMachineSettings cncSettings_;
     std::vector<int> rotationAxesOrderMap_ = {0, 1, 2}; // mapping axis sequence number to axis number in storage
```

Comment:
isn't this order part of cncSettings_?

Reply:
this is an auxiliary structure. in fact, it can be deleted